elements/lisk-genesis/src/schema.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { Schema } from '@liskhq/lisk-codec';
+
+import {
+	blockSchema,
+	baseAccountSchema,
+	blockHeaderSchema,
+} from '@liskhq/lisk-chain';
+import { mergeDeep } from './utils';
+import { EMPTY_BUFFER, EMPTY_HASH } from './constants';
+
+export const genesisAccountSchema = baseAccountSchema;
+export const genesisBlockSchema = mergeDeep({}, blockSchema, {
+	properties: {
+		payload: {
+			minLength: 0,
+			maxLength: 0,
+		},
+	},
+}) as Schema;
+
+export const genesisBlockHeaderSchema = mergeDeep({}, blockHeaderSchema, {
+	properties: {
+		height: {
+			minimum: 0,
+		},
+		version: {
+			const: 0,
+		},
+		generatorPublicKey: {
+			const: EMPTY_BUFFER,
+		},
+		reward: {
+			const: BigInt(0),
+		},
+		signature: {
+			const: EMPTY_BUFFER,
+		},
+		transactionRoot: {
+			const: EMPTY_HASH,
+		},
+	},
+}) as Schema;
+
+export const genesisBlockHeaderAssetSchema = {
+	$id: '/genesis_block/header/asset',
+	type: 'object',
+	required: ['accounts', 'initDelegates', 'initRounds'],
+	properties: {
+		accounts: {
+			type: 'array',
+			items: {
+				...genesisAccountSchema,
```

Comment:
base account schema doesn't have the asset schema. It should be obtained from outside of function, and combine them together, as in chain.ts